Common/UnitDefinitions/Density.json

```diff
@@ -189,6 +189,18 @@
           "Abbreviations": [ "ppg (imp.)" ]
         }
       ]
+    },
+    {
+      "SingularName": "KilogramPerLiter",
+      "PluralName": "KilogramsPerLiter",
+      "FromUnitToBaseFunc": "x*1e3",
```

Comment:
@angularsen just a note when we add new units, we should make sure BaseUnits are defined.

Reply:
@tmilnthorp Can you provide an example? It was my understanding that BaseUnits are defined at the top of the file (KilogramPerCubicMeter in this case).

Reply:
Those are BaseDimensions (for example dimensions of area is Length: 2).

In my recent PR #625, you can see an example for Density [here](https://github.com/angularsen/UnitsNet/pull/625/files#diff-c93aba4c1b33543ddb186a74146b8324).

Reply:
In this case, it's Kilogram and Decimeter

Reply:
@jbrandin This is a very recent addition to the library so it's quite understandable you didn't catch this :-) It will be easier once we have populated all the units with these base units. I will update the wiki with the extra step.

Reply:
Completely forgot about the Wiki. Ah doc. My weakness 😞

Reply:
Ah, very recent indeed!

```suggestion
      "BaseUnits": {
        "L": "Decimeter",
        "M": "Kilogram"
      },
      "FromUnitToBaseFunc": "x*1e3",
```

Reply:
Regen and you'll be good to go!